src/test/java/skyler/main/ParserTest.java

```diff
@@ -0,0 +1,24 @@
+package skyler.main;
+
+import org.junit.jupiter.api.Test;
+import skyler.exception.SkylerException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ParserTest {
+
+    @Test
+    public void processUserInput_eventCommand_addsEventTask() throws SkylerException {
```

Comment:
Expected behaviour in the naming of the test method could be in past tense? I.e. processUserInput_eventCommand_eventTaskAdded. I am not sure about this but the examples shown are given as such.